user-stories/US-07-Privacy-request-capture-service.md

```diff
@@ -0,0 +1,300 @@
+# Privacy Request capture service
+
+<!-- prettier-ignore -->
+| User Stories | 07 |
+| ---------- | ---- |
+| **Status** | Draft |
+| **Design** | [Figma activity diagram (draft)](https://www.figma.com/file/miUd9PEmLrjut53rwrQViX/Data-rights-request-capture-service?node-id=0%3A1)
+| **Previous discussions** | https://github.com/blindnet-io/product-ideas/issues/22
+| **Other documents** | [Privacy Request Interchange Vocabulary](../refs/schemas/priv/RFC-PRIV.md), [Examples of Privacy Requests](../refs/schemas/priv/examples.md), [Scenarios](../refs/schemas/priv/scenarios.md), [Expected Behavior of Systems](../refs/schemas/priv/expected-behavior.md)
+
+## Introduction
+
+Privacy Request Capture Service is a component of the [High Level Architecture](../refs/high-level-architecture). The user stories listed here illustrate its behavior, but are not to be considered exhaustive with regards to all the functionalities this component SHOULD offer.
+
+When this component is installed by a System and used to interact with the Internet Users of that System, we call that **Local** Privacy Request Capture Service.
```

Comment:
I reshuffled US into the following sections : 
07-1 Common US for Local and Global Privacy Request Capture Service
07-2 US specific to Local Privacy Request capture service
07-3 US specific to Global Privacy Request Capture Service
TBD : 07-4 US related to a particular data capture : Common or Global or Local ? 
TBD: 07-5 Out of Privacy Request Capture Service scope ? 
Please have a look and let me know what you think

Reply:
- I deleted the section "TBD : 07-4 US related to a particular data capture : Common or Global or Local ?" and moved the 2 US from there to section "07-2 US specific to Local Privacy Request capture service" 
- @filipblnt what do you think I delete the "TBD: 07-5 Out of Privacy Request Capture Service scope ?" section and kept the US in my notes in case we need it later ?